[2023-03-07 14:38:59,888] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.Executa_api scheduled__2023-03-06T01:05:00+00:00 [queued]>
[2023-03-07 14:38:59,892] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.Executa_api scheduled__2023-03-06T01:05:00+00:00 [queued]>
[2023-03-07 14:38:59,892] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-07 14:38:59,892] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-07 14:38:59,892] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-07 14:38:59,901] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): Executa_api> on 2023-03-06 01:05:00+00:00
[2023-03-07 14:38:59,905] {standard_task_runner.py:52} INFO - Started process 64692 to run task
[2023-03-07 14:38:59,907] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'meu_primeiro_dag', 'Executa_api', 'scheduled__2023-03-06T01:05:00+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/cotacao_dolar.py', '--cfg-path', '/tmp/tmpev51wpa4', '--error-file', '/tmp/tmpw6xmxu7v']
[2023-03-07 14:38:59,907] {standard_task_runner.py:80} INFO - Job 97: Subtask Executa_api
[2023-03-07 14:38:59,930] {task_command.py:370} INFO - Running <TaskInstance: meu_primeiro_dag.Executa_api scheduled__2023-03-06T01:05:00+00:00 [running]> on host veripc-System-Product-Name
[2023-03-07 14:39:00,039] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=meu_primeiro_dag
AIRFLOW_CTX_TASK_ID=Executa_api
AIRFLOW_CTX_EXECUTION_DATE=2023-03-06T01:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-06T01:05:00+00:00
[2023-03-07 14:39:00,039] {python.py:173} INFO - Done. Returned value was: None
[2023-03-07 14:39:00,042] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=meu_primeiro_dag, task_id=Executa_api, execution_date=20230306T010500, start_date=20230307T173859, end_date=20230307T173900
[2023-03-07 14:39:00,079] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-07 14:39:00,099] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
